catroid/src/main/java/org/catrobat/catroid/ui/dialogs/AcceptTermsOfUseDialogFragment.java

```diff
@@ -61,6 +64,17 @@ public void onClick(DialogInterface dialog, int id) {
 				}).create();
 		termsOfUseDialog.setCanceledOnTouchOutside(true);
 
+		termsOfUseDialog.setOnShowListener(new DialogInterface.OnShowListener() {
+			@Override
+			public void onShow(DialogInterface dialog) {
+				if (getActivity() == null) {
+					Log.e(DIALOG_FRAGMENT_TAG, "onShow() Activity was null!");
+					return;
+				}
+				TextSizeUtil.enlargeViewGroup((ViewGroup) termsOfUseDialog.getWindow().getDecorView().getRootView());
+			}
+		});
+
```

Comment:
No it isn't, just noticed it (I've removed it in my branch in the meantime anyway.. so just keep it)